serverless/pages/spaces.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: /serverless/spaces
+title: Spaces
+description: Organize your project and objects into multiple spaces.
+---
+
+This content applies to: <DocBadge template="elasticsearch" slug="/serverless/elasticsearch/what-is-elasticsearch-serverless" /> <DocBadge template="security" slug="/serverless/security/what-is-security-serverless" />
+
+Spaces enable you to organize your dashboards and other saved
+objects into meaningful categories. Once inside a space, you see only
+the dashboards and saved objects that belong to that space.
+
+When you create and enter a new project, you're using the default space of that project.
+
+You can identify the space you're in or switch to a different space from the header.
+
+<DocImage size="l" url="../images/space-breadcrumb.png" alt="Space breadcrumb" />
+
+You can view and manage the spaces of a project from the **Spaces** page in **Management**.
+
+## Required permissions
+
+You must have the admin role on the project to manage its **Spaces**.
```

Comment:
Maybe "an admin role" instead of "the admin role"? Security has more than one admin-level role in its predefined roles, and I assume that the new serverless custom roles will allow customers to create multiple admin-level roles, too.
```suggestion
You must have an admin role on the project to manage its **Spaces**.
```